src/ir/function.rs

```diff
@@ -377,9 +377,14 @@ impl FunctionSig {
             return Err(ParseError::Continue);
         }
 
-        // Don't parse operatorxx functions in C++
         let spelling = cursor.spelling();
-        if spelling.starts_with("operator") {
+
+        // Don't parse operatorxx functions in C++
+        let is_operator = |spelling: &str| {
```

Comment:
I would've just inlined this condition, but sure :-)

Reply:
You're right, it would have been more consistent with the prevailing style to inline it. I tend to think that finding a good name like `is_operator` helps to show that a package of logic is coherent whole, rather than an amalgamation of separate conditions.

I will remember it for next time. :)